packages/p2p/src/components/buy-sell/buy-sell-row.scss

```diff
@@ -36,7 +36,7 @@
 
         @include mobile {
             display: grid;
-            grid-template-columns: 3fr 1fr;
+            grid-template-columns: 3fr 1.7fr;
```

Comment:
Set the button column to be `1.7fr` since with less than `1.7fr`, the button width would wrap the button text for ES and ST language, and after consulting with design team they wanted to avoid the text to be wrapped if possible, and only values `>= 1.7fr` would the text for ES and ST language not be wrapped:

### With 1.6fr:
<img width="459" alt="Screenshot 2022-07-18 at 3 37 10 PM" src="https://user-images.githubusercontent.com/103016120/179464511-fcd044c6-c33c-42de-a287-488b3cd9327d.png">

### With 1.7fr:
<img width="446" alt="Screenshot 2022-07-18 at 3 37 00 PM" src="https://user-images.githubusercontent.com/103016120/179464536-7d82b58e-d619-4cfa-b993-4b5716f0760b.png">